PickWords/Source/framework.h

```diff
@@ -1,5 +1,11 @@
 #pragma once
 
 #define WIN32_LEAN_AND_MEAN             // Exclude rarely-used stuff from Windows headers
+
+
+#ifdef _WIN32
 // Windows Header Files
 #include <windows.h>
```

Comment:
not sure whats used from the windows header

Reply:
Can you please move the LEAN AND MEAN into the _WIN32 if statement.

The windows header is probably used for the shared library and timer functionality. Windows code won't build without it.